keyboards/primekb/prime_m/rules.mk

```diff
@@ -14,22 +14,22 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
-MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no        # Console for debug
-COMMAND_ENABLE = no        # Commands for debug and configuration
+BOOTMAGIC_ENABLE = no		# Virtual DIP switch configuration
```

Comment:
Could you please change all these tabs back to spaces?

Reply:
Done